### PR TITLE
Update UG for sort and list

### LINKED ISSUE #132: Update UG for sort and list

### PR DESCRIPTION
**Pull request by seah-minlong**
Fixes #132

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -94,16 +94,26 @@ Examples:
...
+
+### Sorting the displayed list: `sort`
+
+Sorts the currently visible list of persons by a specified field. 
+
+Format: `sort [s/SORT_OPTION]`
+
+* The sort order will remain active across future searches or filters until a new sort option is applied or the list is sorted again.
+* If no sort option is provided, calling `sort` will restore the list to its original order (i.e., the order in which contacts were added).
+* This command only changes the order of the visible contacts without affecting any filters or searches that are currently applied.
```
Comment (Justin-Yeo) [inline docs/UserGuide.md:107]: I get what you mean for this line (idea about how we can search and then apply sort on the returned search results). However, I am wondering if there is anyway we can make it more straightforward? (Maybe explicitly state that sort commands can be applied on filtered/search results)

**Review by Justin-Yeo — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (seah-minlong) [reply docs/UserGuide.md:107]: Would this be better?

* You can use `sort` to arrange the contacts in the list you're currently viewing, whether it's the full list or a subset from commands like `search` or `viewGroup`.
* If no sort option is provided, `sort` will restore the list to its original order (i.e., the order in which contacts were added).
* The sort order remains active until you apply a new sort option or reset the list.

Comment (Justin-Yeo) [reply docs/UserGuide.md:107]: yup sounds great!

Comment (seah-minlong) [reply docs/UserGuide.md:107]: Sounds good!

**Review by Justin-Yeo — APPROVED** [review]
LGTM!

**Pull request merged**